Cargo.toml

```diff
@@ -13,9 +13,22 @@ readme = "./README.md"
 [dependencies]
 trustfall = "0.7.1"
 rustdoc-types = "0.29.1"
+rayon = { version = "1.10.0", optional = true, default-features = false }
+
+[features]
+default = ["rayon"]
+rayon = ["dep:rayon"]
```

Comment:
We'd like a fair bit of documentation to be available here, in order to help downstream maintainers decide if this feature is right for their use case.

We specifically want to describe what behavior changes this feature produces, and what it currently might not do but might start doing in the future.

Reply:
Will do! (also, the default to `rayon` is temporary for ease of development)